### DISCUSSION TITLE
一些小建议

### DISCUSSION THREAD

**Discussion by FelixXiong** [WatsonYao/nian_app#59] [Q&A] [answered]
目前我在找Android上的日记App，直到我发现了这个。

以下是一些建议：
- 支持导入其他App的内容和媒体
- 支持富文本/Markdown编辑
- 支持更改时间（目前手动从别的app复制粘贴，无法更改那天的时间）
- 插入图片时，复制一份到自己的文件夹（为了原片删除而nian可以保留）

**Comment by WatsonYao** [comment, marked as answer]
感谢使用和反馈，讨论如下
支持导入其他App的内容和媒体
---- 无此类计划 
支持富文本/Markdown编辑
---- 无此类计划
支持更改时间（目前手动从别的app复制粘贴，无法更改那天的时间）
---- 可使用进展补录
插入图片时，复制一份到自己的文件夹（为了原片删除而nian可以保留）
---- 没看明白 T_T
